iwyu_location_util.h

```diff
@@ -37,8 +37,8 @@
 // file path, line number, column number, and which file (if any)
 // #includes the file.
 
-#ifndef DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_IWYU_FILE_UTIL_H_
-#define DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_IWYU_FILE_UTIL_H_
+#ifndef INCLUDE_WHAT_YOU_USE_IWYU_LOCATION_UTIL_H_
+#define INCLUDE_WHAT_YOU_USE_IWYU_LOCATION_UTIL_H_
```

Comment:
This is already fixed on master, can you rebase?

$ git add remote upstream git@github.com:include-what-you-use/include-what-you-use.git
$ git checkout master
$ git pull upstream master
$ git checkout ClangTidyWarningsCleanup
$ git rebase master

# resolve conflicts...